inc/taimodule.h

```diff
@@ -60,33 +60,11 @@
 typedef enum _tai_module_oper_status_t
 {
     TAI_MODULE_OPER_STATUS_UNKNOWN,             /**< Unknown */
-    TAI_MODULE_OPER_STATUS_RESET,               /**< Reset */
     TAI_MODULE_OPER_STATUS_INITIALIZE,          /**< Initialize */
```

Comment:
Can you explain what the "initialize" state for a module is? I would have thought that "reset" state would have been a module state, not "initialize".

In my thinking, selecting the "reset" state causes the module to be held in reset, via an external pin or some software initiated mechanism. Selecting the "ready" state causes the reset to be removed and the module is allowed to operate.

Reply:
I'm thinking about adding `module_admin_status` as R/W attribute and make `module_oper_state` as read-only attribute like below.

https://github.com/openconfig/public/blob/master/release/models/interfaces/openconfig-interfaces.yang#L436-L505

'initialize' state if a state where a module initializes peripheral devices on it before going to 'ready' state.

Reply:
I like the separating out the r/w module_admin_state and the r/o module_oper_state.

Reply:
I'll create another PR for admin/oper state. Merging this PR as is for now.